style.css

```diff
@@ -0,0 +1,101 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+*{
+    font-family: 'Montserrat', sans-serif;
+}
+body{
+    background-color: #000000;
+}
+.card-area{    
+    background: linear-gradient(98.76deg, #1C0E0E 24.39%, #1C1833 81.33%);
+    border-radius: 50px;
+}
+/* NAVBAR */
+nav{display: flex;
+    padding: 45px 150px;
+    margin-left: 450px;
+}
+nav #marvel-logo{
+    width: 150px;
```

Comment:
I would recommend you to use the font-sizes with rems

Reply:
Changed the lines 28 and 75 from 'font-size:30px;' to 'font-size:1.875rem;'. 
Likewise, changed lines 96 and 103 from 'font-size: 28px;' to 'font-size: 1.75rem;' and the line